docs/standard/frameworks.md

```diff
@@ -1,56 +1,293 @@
 ---
-title: Frameworks and Targets
-description: Explains the concepts of framework targets when writing .NET code.
-keywords: .NET, .NET Core
+title: Target frameworks
+description: Explains the concepts of target frameworks when writing .NET code.
+keywords: .NET, .NET Core, framework, tfm
 author: richlander
 ms.author: mairaw
-ms.date: 09/19/2016
+ms.date: 03/13/2017
 ms.topic: article
 ms.prod: .net
 ms.technology: dotnet-standard
 ms.devlang: dotnet
 ms.assetid: 6ef56a2e-593d-497b-925a-1e25bb6df2e6
 ---
 
-# Frameworks and Targets
+# Target frameworks
 
-The .NET ecosystem has a concept of frameworks. Frameworks define the API that you can use to target a particular platform. The .NET Framework 4.6 is one of those platforms. Frameworks are used in Visual Studio and other IDEs and editors to provide you with the correct set of APIs. They are also used by NuGet, for both production and consumption of NuGet packages, to ensure that you produce and use appropriate packages (and underlying assets) for the framework you are targeting. One can think of frameworks as one of the key currencies in the .NET ecosystem. The concept is there for correctness, to help you and your customers avoid seeing @System.MissingMethodException and friends at runtime.
+The .NET ecosystem has a concept of *target frameworks*. Frameworks define the API that you can use to target a particular platform. For example, the .NET Framework 4.6 is a platform that you can target. Visual Studio and other IDEs and editors use frameworks to provide you with the correct set of APIs. NuGet also uses them to ensure that you produce and consume appropriate NuGet packages and underlying assets for the framework you're targeting.
 
-## Framework Versions
+Refernece | Name
```

Comment:
typo: Refernece

also, this feels a bit lost here without the explanation we had before.